_announcements/2016-05-05-mesh101.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "mesh.101 workshop"
+dateUntil: 2016-05-05
```

Comment:
I think we want that extra frontmatter for events? I am imagining an announcement could be like as simple as "Join us every tuesday" or "looking forward to seeing you at x"